nuimo.py

```diff
@@ -307,7 +316,7 @@ def write_matrix(self, matrix, timeout, brightness=1.0):
         led_data = bytearray(matrix_bytes_list)
         led_uuid = self.characteristics_by_name['LED_MATRIX']['uuid']
         led_handle = self.value_handle(led_uuid)
-        self.write_by_handle(led_handle, bytes(led_data))
+        self.write_by_handle_async(led_handle, bytes(led_data), response)
```

Comment:
Our mobile apps use our Nuimo SDKs for mobile platforms (open sourced, too). These SDKs are always up to date since they power the Nuimo apps. In their case we do wait for the matrix write response and don't send any further request until we received the response. SDK users can however specify to make GATT write requests without responses.